taxa/Lupinus/index.html

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Pages generated by a plugin should not be included in the repository. The plugin should generate them at compile time and include them in the generated site located in the _site directory.